types/power-assert/ts3.7/index.d.ts

```diff
@@ -0,0 +1,36 @@
+// Type definitions for power-assert 1.5.1
+// Project: https://github.com/twada/power-assert
+// Definitions by: vvakame <https://github.com/vvakame>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// copy from assert external module in node.d.ts
+
+import * as module from "../index";
+
+export = assert;
+export as namespace assert;
+
+declare function assert(value: any, message?: string): asserts value;
+declare namespace assert {
+    export import AssertionError = module.AssertionError;
+
+    export import fail = module.fail;
+    export import ok = module.ok;
```

Comment:
Would it make more sense to have `ok` be the following?
```suggestion
    export function ok(value: any, message?: string): asserts value;
```

Reply:
done.